xbmc/platform/win32/powermanagement/Win32DPMSSupport.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *      Copyright (C) 2009-2013 Team XBMC
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "Win32DPMSSupport.h"
+
+#include "rendering/dx/RenderContext.h"
+#include "ServiceBroker.h"
+#include "utils/log.h"
+#include "windowing/GraphicContext.h"
+
+void CWin32DPMSSupport::Register()
```

Comment:
Need to add call to this method here https://github.com/xbmc/xbmc/blob/master/xbmc/windowing/windows/WinSystemWin32.cpp#L85

Reply:
hehe, I said in the PR that I didn't register windows or osx yet but I'll add it in here :)